rqt_py_common/src/rqt_py_common/message_tree_model.py

```diff
@@ -70,13 +82,14 @@ def _recursive_create_items(self, parent, slot, slot_name, slot_type_name, slot_
             row.append(item)
 
         is_leaf_node = False
-        # TODO(mlautman): Work around missing _slot_types in new msg types
```

Comment:
I actually fixed this with https://github.com/ros2/rosidl_python/pull/19 

I thought I had told you about that but maybe not... We need a good system for communicating things like this. Also we should start sharing the repos file for testing so that we are both building/testing against the same ros2 env